pythermalcomfort/classes_input.py

```
@@ -768,6 +774,36 @@ def __init__(
         )
 
 
+@dataclass
+class GaggeTwoNodesJiInputs(BaseInputs):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)